llpc/tool/amdllpc.cpp

```diff
@@ -1499,36 +1499,63 @@ static Result processPipeline(ICompiler *compiler, ArrayRef<std::string> inFiles
 #ifdef WIN_OS
 // =====================================================================================================================
 // Finds all filenames which can match input file name
-void findAllMatchFiles(const std::string &inFile,           // [in] Input file name, include wildcard
-                       std::vector<std::string> *pOutFiles) // [out] Output file names which can match input file name
-{
+//
+// @param       inFil    : Input file name, including a wildcard.
+// @param [out] outFiles :  Output vector with matching filenames.
+static void findAllMatchFiles(const std::string &inFile, std::vector<std::string> *outFiles) {
   WIN32_FIND_DATAA data = {};
 
-  // Separate folder name
+  // Separate folder name.
   std::string folderName;
   auto separatorPos = inFile.find_last_of("/\\");
-  if (separatorPos != std::string::npos) {
+  if (separatorPos != std::string::npos)
     folderName = inFile.substr(0, separatorPos + 1);
-  }
 
-  // Search first file
-  HANDLE pSearchHandle = FindFirstFileA(inFile.c_str(), &data);
-  if (pSearchHandle == INVALID_HANDLE_VALUE) {
+  // Search first file.
+  HANDLE searchHandle = FindFirstFileA(inFile.c_str(), &data);
+  if (searchHandle == INVALID_HANDLE_VALUE)
     return;
-  }
 
-  // Copy first file's name
-  pOutFiles->push_back(folderName + data.cFileName);
+  // Copy first file's name.
+  outFiles->push_back(folderName + data.cFileName);
 
-  // Copy other file names
-  while (FindNextFileA(pSearchHandle, &data)) {
-    pOutFiles->push_back(folderName + data.cFileName);
-  }
+  // Copy other file names.
+  while (FindNextFileA(searchHandle, &data))
+    outFiles->push_back(folderName + data.cFileName);
 
   FindClose(pSearchHandle);
```

Comment:
@kuhar , this line breaks windows build, could  you change pSearchHandle to searchHandle?

Reply:
Done. Thanks for catching.